src/python/grpcio/grpc/_cython/BUILD.bazel

```diff
@@ -2,13 +2,21 @@ package(default_visibility = ["//visibility:public"])
 
 load("//bazel:cython_library.bzl", "pyx_library")
 
+genrule(
+    name = "copy_roots_pem",
+    srcs = ["//:etc/roots.pem"],
+    outs = ["_credentials/roots.pem"],
+    cmd = "cp $(SRCS) $(@)",
+)
+
 pyx_library(
     name = "cygrpc",
     srcs = glob([
         "**/*.pxi",
         "cygrpc.pxd",
         "cygrpc.pyx",
     ]),
+    data = [":copy_roots_pem"],
```

Comment:
nit: We have credentials in https://github.com/grpc/grpc/tree/master/src/python/grpcio_tests/tests/unit/credentials. We could reuse them. It might be complex, and it is okay to be complete in separate PR.

Reply:
Those are test credentials. Note that the directory says that they're not meant for production. Not that this change is to the production `cygrpc` target. We needed this in order to support creds callbacks in Bazel since the beginning. This flaw just wasn't noticed until xDS started requiring it.

Sorry for resurrecting this thread 24 days later. I just now got back onto this PR.